samples/tfm_integration/tfm_ipc/CMakeLists.txt

```
@@ -14,20 +14,19 @@ elseif (CONFIG_TFM_BL2_FALSE)
 endif()
 
 # Add "tfm" as an external project via the TF-M module's cmake file
-if(${TFM_TARGET_PLATFORM} STREQUAL "STM_NUCLEO_L552ZE_Q")
+if(${TFM_TARGET_PLATFORM} STREQUAL "stm/nucleo_l552ze_q")
```

Just noticing, using `${TFM_TARGET_PLATFORM}` without qoutes is dangerous in case `TFM_TARGET_PLATFORM` has not been defined.
Will give an error like:
```
CMake Error at samples/tfm_integration/tfm_ipc/CMakeLists.txt:17 (if):
  if given arguments:

    "STREQUAL" "stm/nucleo_l552ze_q"

  Unknown arguments specified
```
whereas qouting would be comparing an empty string, which would be a valid comparison.
```suggestion
if("${TFM_TARGET_PLATFORM}" STREQUAL "stm/nucleo_l552ze_q")
```
and of cause be false, if `TFM_TARGET_PLATFORM` os empty/undefined.

Note, as this is old code (only changing right side string), then it has probably not been a problem up til noew and can thus be cleaned up later.

---

Thanks @tejlmand , this is good feedback. I can actually fix this in the current PR.

---

Fixed now.